zigpy/zcl/clusters/general.py

```diff
@@ -448,14 +462,15 @@ class LevelControl(Cluster):
         0x4000: ("start_up_current_level", t.uint8_t),
     }
     server_commands = {
-        0x0000: ("move_to_level", (t.uint8_t, t.uint16_t), False),
+        0x0000: ("move_to_level", (t.uint8_t, t.uint16_t), False),  # + 2 optionals
         0x0001: ("move", (t.uint8_t, t.uint8_t), False),
         0x0002: ("step", (t.uint8_t, t.uint8_t, t.uint16_t), False),
         0x0003: ("stop", (), False),
         0x0004: ("move_to_level_with_on_off", (t.uint8_t, t.uint16_t), False),
         0x0005: ("move_with_on_off", (t.uint8_t, t.uint8_t), False),
         0x0006: ("step_with_on_off", (t.uint8_t, t.uint8_t, t.uint16_t), False),
         0x0007: ("stop", (), False),
+        # 0x0008: ("move_to_closest_frequency", (t.uint16_t), False),
```

Comment:
Do we need to add the commented out attribute?

Reply:
The commented line is a command, but when I added it the tests didn't pass and I did not understand why not.

So I commented it out and made [a note about it](https://github.com/zigpy/zigpy/pull/905#issuecomment-1030983815)

Reply:
A one-tuple containing just `t.uint16_t` would be `(t.uint16_t,)`.

I've just merged https://github.com/zigpy/zigpy/pull/716, which rewrites just about every single ZCL command definition and likely fixes many of the problems you pointed out.

Reply:
Thank you for explaining the tuple subtlety.  

Ok, so this PR is useless and I can close it and remove the branch?  (just adding a last change to test the tuple fix ).

Reply:
that was a good catch. @mdeweerd are you running pre-commit hooks? i thought those were supposed to catch and correct this, or at least modify it.

Reply:
I have not set up pre-commit hooks, I have just set up "tox" that I run manually to debug cases that pop up.

Now I installed: 
- `pip install pre-commit`
- and, in the repository:  `pre-commit install`
- and, to test: `pre-commit run --all-files`

So that should avoid some commits with failing tests in the future.